pages/people/[id]/edit/[[...stepId]].tsx

```diff
@@ -39,6 +39,7 @@ const UpdatePerson = (): ReactElement => {
       contextFlag: formData.contextFlag || user.permissionFlag,
       nhsNumber: Number(formData.nhsNumber),
       createdBy: user.email,
+      restricted: formData.restricted || person?.restricted,
```

Comment:
Can we have a situation when formData object doesn't have a restricted key and also person is undefined?